project/kevinshlo/README.md

```diff
@@ -0,0 +1,86 @@
+# Corner Stitching
+
+312551134 羅時弘
+
+## Basic Information
+
+"Corner Stitching" is a fundamental data structure for tile-based detailed routing in EDA.  
+The code of this project is hosted on GitHub [kevinshlo/CornerStitching](https://github.com/kevinshlo/CornerStitching) .
+
+## Problem to Solve
+
+In electronic design automation (EDA), the last step of circuit design is detailed routing, which is considered the most difficult and time consuming step. There are several categories of methods to address this problem, and one of them is called tile-based detailed routing. In which, corner stiching is a fundamental data structure to represent tiles on routing layers.
```

Comment:
It is bad to use one line for a single paragraph, for it makes the lines to be too long. It is hard to add comments at specific location for a long line. Please change to 79-char width.